### DISCUSSION TITLE
why my 4060lp do not work?

### DISCUSSION THREAD

**Discussion by guipeidanggou** [nagadomi/nunif#249] [Q&A]
at first, it does work, but now jt just use my cpu, i am sure that i didnt change anything

**Comment by guipeidanggou** [comment]
![QQ20241103-000632](https://github.com/user-attachments/assets/f4f6016f-4f2b-4893-a87c-2df588e5d184)
![Uploading 屏幕截图 2024-11-03 000545.png…]()

**Reply by nagadomi** [reply]
Make sure you select `NVIDIA GeForce RTX 4060` instead of `CPU` for `Device` field.

**Reply by guipeidanggou** [reply]
sure，in fact what ever i chose,now it only use cpu
![Uploading 屏幕截图 2024-11-03 121514.png…]()

**Reply by guipeidanggou** [reply]
![屏幕截图 2024-11-03 121514](https://github.com/user-attachments/assets/2505b890-7993-4d6a-b039-2f826e99da54)

**Reply by guipeidanggou** [reply]
![屏幕截图 2024-11-03 122254](https://github.com/user-attachments/assets/1aa4379d-af16-4751-862a-03231ab2ef35)
so weird

**Reply by nagadomi** [reply]
On Windows, it is very slow when Shared GPU Memory(共享GPU内存) is used. If it cannot be reduced to 0, try rebooting the system once.
https://www.reddit.com/r/StableDiffusion/comments/x308i4/comment/immq838/

Also, in the case of Laptop PC, make sure it is not in power save mode.
output to USB drive or network drive will also slow down the process.

**Reply by guipeidanggou** [reply]
i try to shut this down, at first every thing were right, but today, it shows this: i can not even transfer a single picture which is only 3mb
![image](https://github.com/user-attachments/assets/969308e1-91c9-4c41-8071-fe9cd2f3c120)
i ve already try : reboot my computer , reinstall everything(python and iw3), but these not work.

**Reply by guipeidanggou** [reply]
in the same time, video function is normal.
![image](https://github.com/user-attachments/assets/8a1cc373-5bfa-47f9-9365-da37eeb53aaf)

**Reply by nagadomi** [reply]
As for out of memory when processing images, it is probably due to the aspect ratio of the input image.
The higher the difference between the height and width size, more VRAM is used.

I plan to limit the maximum size of depth.
At the moment, workarounds for this problem are listed below.
- Use a model that uses less VRAM, such as Any_B or Any_S
- Try `Depth Resoltuion`=252 or 192 (392 by default. Depth Resolution for Depth-Anything is the size on the small side)
- Avoid using images with extreme aspect ratios

**Reply by guipeidanggou** [reply]
really thanks！now it is working well.
